code/modules/reagents/reagent_containers/glass/beaker.dm

```diff
@@ -110,7 +110,7 @@
 	desc = "A small plastic vial."
 	icon_state = "vial_plastic"
 	matter = list(MATERIAL_PLASTIC = 1)
-
+	spawn_blacklisted = TRUE
```

Comment:
I prefer that you use spawn_tags = null
or
spawn_frequency = 0.

Reply:
you could do: spawn_tags = SPAWN_TAG_ITEM. for this to appear in the low cost item spawner.

Reply:
I mean I guess I can just do 0 spawn frequency if that applies to children as well, but like, any particular reason why?

Reply:
I'll do spawn_tags = null actually. anyway, done.